helm/hpcc/templates/thor.yaml

```diff
@@ -13,7 +13,7 @@
 {{- $eclAgentScope := dict "name" $eclAgentName "useChildProcesses" $eclAgentUseChildProcesses "replicas" $eclAgentReplicas | merge (pick . "keepJobs") }}
 {{- $thorAgentScope := dict "name" $thorAgentName "replicas" $thorAgentReplicas  | merge (pick . "keepJobs") }}
 {{- $hthorScope := dict "name" $hthorName | merge (pick . "multiJobLinger") }}
-{{- $thorScope := omit . "eclagent" "thoragent" "hthor" "logging" "eclAgentResources" "managerResources" "workerResources" "eclAgentUseChildProcesses" "eclAgentReplicas" "thorAgentReplicas" "eclAgentType" }}
+{{- $thorScope := omit . "eclagent" "thoragent" "hthor" "logging" "eclAgentUseChildProcesses" "eclAgentReplicas" "thorAgentReplicas" "eclAgentType" }}
```

Comment:
I think "eclAgentResources" is not needed (should still be omitted)

Reply:
unrelated, but I think some of these omitted fields are legacy and should be removed (e.g. "eclagent" "thoragent").
As I say unrelated, I'll check and open a separate JIRA